docker-compose.yml

```diff
@@ -7,42 +7,44 @@ services:
     build:
       context: .
       dockerfile: Dockerfile-dev
-    command: script/docker-app.sh
+    command: docker/dev/run.sh
     environment:
-      DB_HOST: ${DB_HOST}
-      DB_USER: ${DB_USER}
-      DB_PASSWORD: ${DB_PASSWORD}
-      DB_NAME: ${DB_NAME}
+      DB_HOST: mysql
+      DB_NAME: portal
+      DB_USER: master
+      DB_PASSWORD: master
       SOLR_HOST: solr
       SOLR_PORT: 8983
-    networks:
-      - appnet
     ports:
-      - "3000:3000"
+      - 3000:3000
     volumes:
       - .:/rigse
-      - "bundle:/bundle"
+      - bundle:/bundle
   solr:
-    image: solr:5.3
-    networks:
-      - appnet
+    image: concordconsortium/docker-solr-portal
     ports:
-      - "8983:8983"
+      - 8983:8983
     volumes:
       - .:/rigse
       - data:/opt/solr/server/solr/mycores
     depends_on:
       - solrdata
-    command: /rigse/docker/start-solr-within-solr-container.sh
+    command: /bin/bash ./start-solr.sh
   solrdata:
-    image: solr:5.3
+    image: concordconsortium/docker-solr-portal
     volumes:
       - data:/opt/solr/server/solr/mycores
     user:
       root
-    command: chown solr:solr /opt/solr/server/solr/mycores
-networks:
-  appnet:
+    command: /bin/bash ./start-volume.sh
+  mysql:
```

Comment:
But I guess this data is persisted as long as you don't remove the container, right? Probably not perfect but I feel good enough in our case. More sophisticated solutions: http://stackoverflow.com/questions/18496940/how-to-deal-with-persistent-storage-e-g-databases-in-docker

For me it the biggest advantage / value of this docker-compose file is that you can spawn a new portal using one command and test something that requires fresh portal instance. I don't think we could develop anything yet due to perf issues with volumes. Or maybe I'm wrong and you use docker for Portal development (since you're on Windows)?

If so, maybe we could use env variables again, but provide default values pointing to mysql container in .env file (hopefully that's possible).